# request message
uint8 robot_position
uint8 motion_type
uint8 rotation_type
uint8 calibration_duration
RGBColor[] rgb_color_list
uint16 camera_width
uint16 camera_height
